wallet/client.go

```diff
@@ -84,3 +86,11 @@ func (c *grpcClient) getTransaction(id tx.ID) (*pactus.GetTransactionResponse, e
 
 	return res, nil
 }
+
+func (c *grpcClient) getFee(amount int64, payloadType payload.Type) (int64, error) {
+	res, err := c.transactionClient.CalculateFee(context.Background(), &pactus.CalculateFeeRequest{Amount: amount, PayloadType: pactus.PayloadType(payloadType)})
```

Comment:
```suggestion
	res, err := c.transactionClient.CalculateFee(context.Background(), &pactus.CalculateFeeRequest{
	Amount: amount, PayloadType: pactus.PayloadType(payloadType)})
```

Break this long line